File: gabriel/Problema0039/Problema0039.js
//Faça um programa que leia e imprima o nome, endereço e telefone de uma pessoa.

const prompt = require('prompt-sync')({sigint: true});

let nome = prompt("Digite seu nome: ");
let endereco = prompt("Digite seu endereco: ");
let telefone = prompt("Digite seu telefone: ");

console.log("Nome: " + nome);
console.log("Endereço: " + endereco);
console.log("Telefone: " + telefone);

File: gabriel/Problema0039/Problema0039.java
//Faça um programa que leia e imprima o nome, endereço e telefone de uma pessoa.

package Problema0039;

import java.util.Scanner;

public class Problema0039 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        
        
        String nome, endereco, telefone;
    
        //entrada
        System.out.println("Digite seu nome: ");
        nome = input.nextLine();
    
        System.out.println("Digite seu endereço: ");
        endereco = input.nextLine();
    
        System.out.println("Digite seu telefone: ");
        telefone = input.nextLine();
    
    
        //saída
        System.out.println("Nome: " + nome);
        System.out.println("Endereço: " + endereco);
        System.out.println("Telefone: " + telefone);
        
      }
}
